src/annotator/config.js

```diff
@@ -47,7 +47,7 @@ function configFrom(window_) {
   //
   // In environments where the config has not been injected into the DOM,
   // we try to retrieve it from the URL here.
-  var directLinkedID = annotationQuery.extractAnnotationQuery(window_.location.href);
+  var directLinkedID = extractAnnotationQuery(window_.location.href);
```

Comment:
Right. The return type could be specified as `{ query: string } | { annotations: string }`.